assets/js/base/hooks/use-spacing-props.ts

```diff
@@ -5,15 +5,15 @@
 import { __experimentalGetSpacingClassesAndStyles } from '@wordpress/block-editor';
 import { isFeaturePluginBuild } from '@woocommerce/block-settings';
 import { isObject } from '@woocommerce/types';
-import { hasSpacingStyleSupport } from '@woocommerce/utils';
 import { parseStyle } from '@woocommerce/base-utils';
 
 type WithStyle = {
 	style: Record< string, unknown >;
 };
 
+// @todo The @wordpress/block-editor dependency should never be used on the frontend of the store due to excessive side and its dependency on @wordpress/components
 export const useSpacingProps = ( attributes: unknown ): WithStyle => {
-	if ( ! isFeaturePluginBuild() || ! hasSpacingStyleSupport() ) {
+	if ( ! isFeaturePluginBuild() ) {
```

Comment:
I believe we should keep checking the spacing style support (i.e., `! hasSpacingStyleSupport()`), even if we still need to find a way to avoid using the `@wordpress/block-editor` dependency!

```suggestion
	if ( ! isFeaturePluginBuild() || ! hasSpacingStyleSupport() ) {
```

Reply:
All this does is check for `__experimentalGetSpacingClassesAndStyles`, and whats more, including an empty `style` prop should not have negative consequences even if included right?

Reply:
@mikejolley, I believe this check was added to [fix crash on WP 5.8](https://github.com/woocommerce/woocommerce-blocks/pull/5515/commits/a17ca36e485d47dac65010ea2599656a8c0f5b47)

Reply:
Thats ok then, we have a minimum version check in place for WP 6.1.